pkgs/by-name/op/openscap/package.nix

```
@@ -0,0 +1,155 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  libsepol,
+  popt,
+  libxml2,
+  libxslt,
+  openssl,
+  util-linux,
+  pcre2,
+  pcre,
+  libselinux,
+  graphviz,
+  glib,
+  python3,
+  swig,
+  python3Packages,
+  libgcrypt,
+  opendbx,
+  xmlbird,
+  haskellPackages,
+  libyaml,
+  xmlsec,
+  bzip2,
+  valgrind,
+  asciidoc,
+  installShellFiles,
+  rpm,
+  system-sendmail,
+  hyperscan,
+  gnome2,
+  curl,
+  procps,
+  systemd,
+  perl,
+  doxygen,
+  pkg-config,
+  perl538Packages,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openscap";
+  version = "1.3.10";
+
+  src = fetchFromGitHub {
+    owner = "OpenSCAP";
+    repo = "openscap";
+    rev = version;
+    hash = "sha256-P7k+Ygz/XmpTSKBEqbyJsd1bIDVJ1HA/RJdrEtjYD5g=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    cmake
+    asciidoc
+    doxygen
+    rpm
+    swig
+    util-linux
+    pkg-config
+  ];
+
+  buildInputs =
+    with perl538Packages;
+    [
+      XMLXPath
+      LinuxACL
+      XMLTokeParser
+    ]
+    ++ [
+      perl
+      popt
+      openssl
+      valgrind
+      pcre2
+      pcre
+      libxslt
+      xmlsec
+      hyperscan
+      libselinux
+      libyaml
+      xmlbird
+      installShellFiles
+      bzip2
+      python3
+      libgcrypt
+      libxml2
+      systemd
+      haskellPackages.pthread
+      graphviz
+      system-sendmail
+      procps
+      asciidoc
+      libsepol
+      curl
+      glib
+      gnome2.ORBit2
+      gnome2.GConf
+      opendbx
+    ];
+
+  prePatch = ''
+    substituteInPlace swig/perl/CMakeLists.txt --replace-fail "DESTINATION ''${PERL_VENDORLIB}" "DESTINATION $out/lib''${PERL_VERSION}"
+    substituteInPlace swig/perl/CMakeLists.txt --replace-fail "DESTINATION ''${PERL_VENDORARCH}" "DESTINATION $out/lib"
+    substituteInPlace src/common/oscap_pcre.c --replace-fail "#include <pcre2.h>" "#include <${pcre2.dev}/include/pcre2.h>"
+    substituteInPlace src/common/oscap_pcre.c --replace-fail "#include <pcre.h>" "#include <${pcre.dev}/include/pcre.h>"
```

```suggestion
    substituteInPlace swig/perl/CMakeLists.txt \
      --replace-fail "DESTINATION ''${PERL_VENDORLIB}" "DESTINATION $out/lib''${PERL_VERSION}" \
      --replace-fail "DESTINATION ''${PERL_VENDORARCH}" "DESTINATION $out/lib"
    substituteInPlace src/common/oscap_pcre.c \
      --replace-fail "#include <pcre2.h>" "#include <${pcre2.dev}/include/pcre2.h>" \
      --replace-fail "#include <pcre.h>" "#include <${pcre.dev}/include/pcre.h>"
```
Why is pcre not properly being added to PATH despite the flags below? This shouldn't then be necessary.